Sources/MapboxMaps/Camera/BasicCameraAnimatorImpl.swift

```diff
@@ -43,7 +43,7 @@ internal final class BasicCameraAnimatorImpl: BasicCameraAnimatorProtocol {
 
     private let mapboxMap: MapboxMapProtocol
 
-    private let mainQueue: MainQueueProtocol
+    private let mainQueue: DispatchQueueProtocol
```

Comment:
should `mainQueue` be renamed (here and below) as well?

Reply:
The instance here is still a main queue, the name of the protocol got updated to better reflect the built-in type it modeled after(`DispatchQueue`).